contrib/stable/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# stable/Dockerfile
+#
+# Build a Skopeo container image from the latest
+# stable version of Skopeo on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=skopeo
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
+
+# Don't include container-selinux and remove
+# directories used by yum that are just taking
+# up space.
+RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install skopeo fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
+
+ADD https://raw.githubusercontent.com/containers/skopeo/master/contrib/skopeoimages/stable/containers.conf /etc/containers/
```

Comment:
That URL does not exist and this PR does not add it AFAICS.

Reply:
Oh fudge, I've left off the skopeoimage directory (and mis-spelled it here), will touch up.